### ISSUE TITLE
Missing dependancy default_ini.h

### ISSUE THREAD

**Issue by misutoneko** [bni/freesynd#1]
Hi,

On Linux Mint (gcc9) the compilation is stopped right away with this:
`dump.cpp:500:10: fatal error: default_ini.h: No such file or directory`

Seems like that file is not listed as a dependancy in src/CMakeLists.txt (line 218 onwards).
Perhaps this is due to changes in CMake (I have CMake 3.16.3, haven't tried it with an older version).

**Comment by bni** [comment]
default_ini.h should be generated during the build by embed executable. embed should be built first and it should look like this:

![Screenshot 2022-11-05 at 11 36 31](https://user-images.githubusercontent.com/422207/200115735-64df60ca-e52b-4e02-98d3-a013907f9b17.png)

I do not know why it isn't for you.

This project is in quite a bad state and it barely works to build and run. I think it needs a lot of work.

**Comment by misutoneko** [comment]
Yes, adding the dependancy will also build embed first and the binary will then be produced.
The SDL2 stuff doesn't seem to be working for me (since using commit f8698e7 works and that's SDL1).
That could well be just something about my setup though, I might take a closer look at it later.

I'm not sure if you're aware but there's been some cleanup work done in here:
https://github.com/CeRiAl/freesynd
But that one hasn't been updated in a while, not sure if it's active.
